qt/applications/workbench/workbench/plotting/figurewindow.py

```diff
@@ -19,7 +19,7 @@
 
 # local imports
 from mantidqt.plotting.figuretype import FigureType, figure_type
-from mantidqt.widgets.common.observing_view import ObservingView
+from mantidqt.widgets.workspacedisplay.observing_view import ObservingView
```

Comment:
Really minor but perhaps `ObservingView` should be outside of `workspacedisplay` now that we are using it elsewhere? That way the packages are not coupled. It could just go to `mantidqt.widgets` ?

Reply:
`ads_observer` and `observing_presenter` are also used outside the `workspacedisplay`, maybe moving them into a `common` module would be better? Or an `observing` module?

Reply:
Yes I think that would be good. Maybe call it `observers` with those 3 classes in? If it expands over time it could become a package later.